packages/tailwindcss/src/variants.ts

```diff
@@ -623,7 +623,12 @@ export function createVariants(theme: Theme): Variants {
   staticVariant('first-line', ['&::first-line'])
 
   // TODO: Remove alpha vars or no?
```

Comment:
Is this still relevant? I don't even know what this comment is trying to say so we might wanna remove it lol